src/cases/NgRestTestCase.php

```diff
@@ -300,5 +351,77 @@ public function beforeTearDown()
         $this->userGroupFixture->cleanup();
         $this->userOnlineFixture->cleanup();
         $this->ngrestLogFixture->cleanup();
+        $this->app->db->createCommand()->dropTable('admin_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_group_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_user_group')->execute();
+    }
+
+    protected function removeApiPermissions()
```

Comment:
protected methods are available trough inheritance, therefore php doc is required including a @since tag.